ote/src/clj/ote/services/routes.clj

```diff
@@ -137,18 +148,44 @@
            (delete! db ::transit/route {::transit/id id})
            id))))
 
+(defn link-interface
+  "This is a helper function for users. Links created route to given service.
+  Users can add interface by hand but this is much more convenient for them."
+  [db user  {is-linked? :is-linked?
```

Comment:
Destructured args are not renamed here, with many args a bit shorter way to do associative destructuring would be using the `:keys` keyword. But a matter of taste.

Reply:
Yes. Valid point, but I don't change it :)